changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 10.3.0 - xxxx-xx-xx =
+* Dev - Removes legacy checkout payment method classes
```

Comment:
Given that we are removing code that developers might be referencing, I think we should flag this as an `Update` as well as adding a section to the Compatibility Notes in readme.txt. We want to make sure people can see the changes quickly before their stores break.
```suggestion
* Update - Remove legacy checkout payment method classes
```